### PR TITLE
[tinygltf] fix json path

### PR DESCRIPTION
**Pull request by fran6co**
This is part of #12199

when using tinygltf with json it will not find it the right place, this fixes that

### CODE REVIEW

File: ports/tinygltf/portfile.cmake [hunk 1]
```diff
@@ -8,8 +8,11 @@ vcpkg_from_github(
...
 # Put the licence file where vcpkg expects it
-file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/tinygltf/LICENSE)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/tinygltf/LICENSE ${CURRENT_PACKAGES_DIR}/share/tinygltf/copyright)
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+
+# Copy the tinygltf header files and fix the path to json
+vcpkg_replace_string(${SOURCE_PATH}/tiny_gltf.h "#include \"json.hpp\"" "#include <nlohmann/json.hpp>")
+file(INSTALL ${SOURCE_PATH}/tiny_gltf.h DESTINATION ${CURRENT_PACKAGES_DIR}/include)
+
+
```
Comment (PhoebeHui) [inline ports/tinygltf/portfile.cmake:17]:
```suggestion
# Copy the tinygltf header files and fix the path to json
vcpkg_replace_string(${SOURCE_PATH}/tiny_gltf.h "#include \"json.hpp\"" "#include <nlohmann/json.hpp>")
file(INSTALL ${SOURCE_PATH}/tiny_gltf.h DESTINATION ${CURRENT_PACKAGES_DIR}/include)

file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
@fran6co, thanks for you PR!

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks for your updates!

**Comment by ras0219-msft** [comment]
Thanks for the fix!

**Pull request merged**